babel.config.js

```diff
@@ -4,12 +4,7 @@ module.exports = {
   ],
   'plugins': [
     '@babel/plugin-proposal-class-properties',
-    [
-      '@babel/plugin-transform-classes',
-      {
-        'loose': true
```

Comment:
This is probably the most significant change to the build. I don't believe it'll break anything, but I'm not fully clear on what the implications of this are.

Leaving loose mode turned on does not fix the bug I mentioned in the description. Disabling it does fix it.